### ISSUE TITLE
Don't show "loading slide" on presentation screen

### ISSUE THREAD

**Issue by projekter** [dannyedel/dspdfviewer#67]
If one navigates quickly through the slides (eg by using a hyperlink which points to a slide far away), it takes a moment until the slide is shown. In this time, a black screen with an information text that the slide is being loaded is presented. This is perfectly fine for the internal view; however, the presentation screen should still show the old slide until the new one is ready to be displayed.

**Comment by dannyedel** [comment]
The problem is twofold. One, a far-away-but-hyperlinked page is not preloaded, that will be fixed soon™ when #60 is done.

Second, the Idea that it does "something" immediatly is important to me: As soon as the system has read my buttonpress, it should provide visual feedback that it understood my command immediatly (so I don't click or press again, for example).

I wouldn't like to simply drop it from the main view (think about the -u0 usecase, where one disables the presenter's view) but if you have an Idea how to UI-design the software communicating "I have received your command, and I'm processing" correctly I could change it.

**Comment by projekter** [comment]
I agree that the user needs feedback. But there are different possibilites:
- If there is no second screen that can differ from the current (maybe because the screen is mirrored), the notification has to be given to the primary screen, so the audience will see it. There's no way to escape this situation.
- If dspdfviewer is allowed to use a second screen, I think that the audience shall never see anything such as a "please wait" message. But of course, the presentation screen needs to be changed in such a way that one knows the presenter doesn't hang.
 - Either (as it is now) blacken the screen and show the text "loading another page" or whatever.
 - Or don't change the screen, but make the mouse curser an hour glass.
 - Or use a different status indicator: I have for example seen a progress bar at the very bottom or top of the screen. This bar does not need to indicate a percentage value, a moving color would be enough.
So feedback yes, but not to the audience.
